index.js

```diff
@@ -104,6 +140,49 @@ class Client {
     }
     return analyses
   }
+
+  /**
+    * Runs MythX analysis return issue information and metadata regarding the run
+    *
+    * @param {options} object - structure which must contain:
+    *      {data} object       - information containing Smart Contract information to be analyzed
+    *      {timeout} number    - optional timeout value in milliseconds
+    *
+    * @returns object which contains:
+    *      (issues}  object - an like object which of issues is grouped by (file) input container.
+    *      {status}  object - status information as returned in each object of analyses().
+    *
+    **/
+  async analyzeWithStatus (options) {
+    const issues = await this.analyze(options)
+    const uuid = issues.uuid
+    delete issues.uuid
+    const status = await this.analyzeStatus(uuid)
+    return {
+      issues,
+      status
+    }
+  }
+
+  async analyzeStatus (uuid, inputApiUrl = defaultApiUrl) {
```

Comment:
I would rename this to `getStatus` or `getAnalysisStatus` ("analyzeStatus" doesn't make sense semantically).

Reply:
Done in 7b66178